test_security/test/test_invalid_secure_node_creation_c.cpp

```diff
@@ -103,14 +104,20 @@ class CLASSNAME (TestInvalidSecureNode, RMW_IMPLEMENTATION) : public ::testing::
     ret = rcl_init_options_init(&init_options, rcl_get_default_allocator());
     ASSERT_EQ(RCL_RET_OK, ret) << rcl_get_error_string().str;
     context = rcl_get_zero_initialized_context();
-    ret = rcl_init(0, nullptr, &init_options, &context);
-    ASSERT_EQ(RCL_RET_OK, ret) << rcl_get_error_string().str;
+    const char * argv[] = {"--ros-args", "--security-context", context_name};
+    ret = rcl_init(3, argv, &init_options, &context);
```

Comment:
@ivanpauno nit: do not hardcode `argv` element count

Reply:
See https://github.com/ros2/system_tests/pull/406/commits/65b46f0f56437040c5feff1676e522e3b2aef5c8.